src/loadsave.jl

```diff
@@ -3,9 +3,11 @@ const sym2saver  = Dict{Symbol,Vector{Symbol}}()
 
 function is_installed(pkg::Symbol)
     isdefined(Main, pkg) && isa(getfield(Main, pkg), Module) && return true # is a module defined in Main scope
-    path = Base.find_in_path(string(pkg)) # hacky way to determine if a Package is installed
-    path == nothing && return false
-    return isfile(path)
+    version = try Pkg.installed(string(pkg)) end # Note: If pkg is registered, but not installed, Pkg.installed() returns nothing. Also, try block returns nothing if something fails.
```

Comment:
I'm pretty sure, that this was the initial version and we dumped it at some point because it was suboptimal. It was either very slow or probably not working in some cases - not 100% sure anymore :(

Reply:
I had a quick flick through the tree. The is_installed method was first introduced by @timholy in commit 2df4c5a6 and the last change or current 3 lines were introduced by you in commit 64c7a728.

There were 3 other commits in between that alter the method in a relevant way:
- 7256f85e
- 0e98cd5e
- 0ec071bd

Prior to the last relevant change, it looks package installation was checked by checking a dictionary for symbols -- I'm too new to Julia to know the context.

Is it that the 3 lines in question are determining the difference between whether a package is registered and installed?

Reply:
Hm maybe I only tried it out locally. Well, it is supposed to tell whether a package can be `used`. Maybe the best way is to mimic what base does when you do `using Module` - I think there should be some functions flying around that look in the load path etc.

Reply:
https://github.com/JuliaLang/julia/blob/10ab2beeb09936ad2cf9d02fce808293e960a519/base/loading.jl#L90 this looks promising :)

Reply:
Oh that isn't on 0.6... https://github.com/JuliaLang/julia/blob/v0.6.1/base/loading.jl#L102 so this uglier method is the one, I guess ;)

Reply:
I admit the find_package method does look more direct.

Reply:
Would you like me to roll back this pull request to 133b993?